src/main/java/racingcar/domain/trynumber/TryNumber.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.domain.trynumber;
+
+public class TryNumber {
+    private static final int MIN_TRY_NUMBER_LIMIT = 1;
+    private static final String ERROR_MORE_THAN_MIN_TRY_NUMBER_MESSAGE = "[ERROR] 시도횟수는 최소 1이상의 값만 입력가능합니다.";
+    public static final int FINISH_VALUE = 0;
+
+    private int tryNumber;
+
+    public TryNumber(int tryNumber) {
+        if (tryNumber < MIN_TRY_NUMBER_LIMIT) {
+            throw new IllegalArgumentException(ERROR_MORE_THAN_MIN_TRY_NUMBER_MESSAGE);
+        }
+
+        this.tryNumber = tryNumber;
+    }
+
+    public boolean isRunnable() {
+        if (tryNumber > FINISH_VALUE) {
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같아요~
```suggestion
return tryNumber-- > FINISH_VALUE;
```

Reply:
알아보기 훨씬 깔끔하고 좋네요. 이런 부분을 왜 놓쳤던 걸까요.. 반드시 기억해야겠네요 👍